doc/network-ng.md

```diff
@@ -40,29 +40,73 @@ other classes.
 
 ## Backend Support
 
-In order to support a new backend, we need to implement a configuration readers and writers. The
-{Y2Network::Sysconfig} module implements support to deal with `sysconfig` files.
+As mentioned above, Y2Network is designed to support different backends. It is expected to implement
+a reader and a writer for each backend (except for AutoYaST, which is an special case). The reader
+will be responsible for checking the system's configuration and building a {Y2Network::Config}
+object, containing interfaces, configurations, routes, etc. On the other hand, the writer will be
+responsible for updating the system using that configuration object.
+
+As a developer, you rarely will need to access to readers/writers because `Yast::Lan` already offers
+an API to read and write the configuration. See the [Accessing the
+Configuration](#accessing-the-configuration) section for further details.
+
+### Sysconfig
+
+The sysconfig backend support is composed by these files:
 
     src/lib/y2network/sysconfig
     ├── config_reader.rb <- READER
     ├── config_writer.rb <- WRITER
-    ├── connection_config_reader_handlers
-    │   ├── eth.rb
-    │   ├── wlan.rb
-    │   └── ...
     ├── connection_config_reader.rb
+    ├── connection_config_readers
+    │   ├── eth.rb
+    │   └── wlan.rb
```

Comment:
maybe add there also that ... as it was before, so it is clear it is not all of them.